python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/radius_proxy.schema.yml

```
@@ -0,0 +1,97 @@
+# Copyright (c) 2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  radius_proxy:
+    type: dict
+    description: |-
+      It forwards RADIUS requests and responses between clients (NAS devices) and RADIUS servers.
+      It acts as a middle layer that helps route, manage, and control authentication,
+      authorization, and accounting (AAA) traffic.
```

This description looks weird - Can we change it to something more descriptive
```suggestion
      Configure RADIUS proxy parameters/
```